conf/messages.en

```diff
@@ -167,7 +167,7 @@ reviewGoods.list.producedInEu = Produced in EU
 reviewGoods.list.destination = Destination
 reviewGoods.list.price = Price paid
 reviewGoods.list.invoice = Invoice number
-reviewGoods.h3 = Do you need to declare more commercial goods?
+reviewGoods.h3 = Do you need to declare more types of goods?
```

Comment:
I think the content is 'more types of commercial goods'

Reply:
This is a cut/paste from the jira ticket. I can have a look at the prototype to see if it has something different

Reply:
@jordanrowe  you are correct, prototype has  "Do you need to declare more types of commercial goods"  I'll try and find out which one is correct

Reply:
Change to version in prototype. David has also updated the ticket.